src/less/link/ds4/link.less

```diff
@@ -4,10 +4,26 @@
 
 @link-nav-color: @color-core-black;
 @link-nav-color-visited: @color-core-black;
-@link-nav-color-hover: @color-link-default;
+@link-nav-color-hover: @color-link-hover;
 @link-action-color: @color-link-default;
 @link-action-color-visited: @color-link-visited;
 @link-fake-color: @color-link-default;
 @link-fake-font-size: @font-size-14;
 
 @import "../base/link.less";
+
+a.action-link {
+    text-decoration: none;
+
+    &:hover {
+        text-decoration: underline;
+    }
+}
+
+button.fake-link--action {
+    text-decoration: none;
```

Comment:
Should we get some accessibility team feed back for this? I believe the reason we have the request for 6.5 to have underlines is because it helps accessibility of our pages.

Yes, that would be a pretty significant change for DS4 users, but it would also mean that we're helping them be just as accessible as DS6 is.

Reply:
Checking with the design team